tests/dummy/mirage/data/models/wedding-application.js

```diff
@@ -5,7 +5,6 @@ export default {
         countryOfBirth: {
           labelAttribute: 'name',
           modelType: 'country',
-          query: {},
```

Comment:
Confused about this change. Isn't having a non-undefined `query` object the signal to fetch this from the store?